packages/stryker/test/unit/StrykerSpec.ts

```diff
@@ -163,7 +166,8 @@ describe('Stryker', function () {
     });
 
     it('should log to have quit early if no mutants were generated', async () => {
```

Comment:
I personally think it would be clearer if we add a separate unit test for the exclusion of mutations. So keep the original test and add one where you explicitly exclude a few. That way it would also be easier to test the new log message.